app/pb/communities.proto

```diff
@@ -68,6 +69,7 @@ message Community {
   uint32 member_count = 10;
   uint32 admin_count = 11;
   uint32 nearby_user_count = 12;
+  int64 thread_id = 13;
 }
```

Comment:
Would be nice to pass the number of comments here too, but could be added later.